### DISCUSSION TITLE
Is there a way to adjust my screen resolution up?

### DISCUSSION THREAD

**Discussion by douglasheld2** [elementary/triage#540] [Q&A] [answered]
I'm using a mid-2010 13" Macbook Pro. (because my project requires an ancient Firewire 800 interface)
I've got Elementary 7.1 installed, and the video card - NVIDIA GeForce 320M - I assume is detected OK. The LCD screen is 1280 x 800.

I chose ElementaryOS largely because the screen real estate was head/shoulders better than Ubuntu or a few other distros I tried.

Here's my problem. I am running a program that provides a dialog box where I can't reach the submit button on the bottom of the window. In the Elementary system settings, I have scaled down the display to as far as it will go - just the hardware range of 1280 x 800.

I know for a fact that if I load up MacOS Mountain Lion, I can get much smaller readable text and be able to reposition and manage a window of this size.

**Question**: Is there something that can be done to further increase screen real estate? Is it possible I need to install an nVidia driver?

**Comment by ryonakano** [comment, marked as answer]
Maybe actually you're using MacBook Pro because MacBook Air has never had FireWire port and there is no mid-2010 model for Air (instead late-2010)?

> **Question**: Is there something that can be done to further increase screen real estate? Is it possible I need to install an nVidia driver?

According to [the Apple's support page](https://support.apple.com/en-us/112604), the LCD of your MacBook Pro only supports up to 1280x800; this limitation wouldn't change if you install NVIDIA driver. So, maybe you may want to consider using an external display for further screen resolution that supports up to 2560x1600.

I guess you can also use the workaround to press Super and drag somewhere of the window to move it and reach the bottom button.

**Reply by douglasheld2** [reply]
Fantastic, thank you. **⌘-drag** was what I needed. I was unaware of this function.
I will also contact the app's author as they documented 1024x768 as their mininum system requirements. I think the modal window in question should have had a vertical scroll bar.

**Comment by jeremypw** [comment]
Note that you can adjust the system text size in System Settings/Appearance.

**Reply by douglasheld2** [reply]
That is helpful, but at a certain point it's the large size of the window decorations that prevents progress. Maybe I'll prepare three screenshots to illustrate the problem.
